tasks/managed/extract-index-image/extract-index-image.yaml

```diff
@@ -35,13 +38,13 @@ spec:
 
         RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/extract-index-image-results.json"
 
-        jsonBuildInfo=$(jq -cr .jsonBuildInfo "$(params.inputDataFile)")
-
-        indexImage=$(echo "$jsonBuildInfo" | jq -cr .index_image)
-        echo -n "$indexImage" > "$(results.indexImage.path)"
+        indexImage="$(jq -cr '.components[].index_image' \
+                    "$(workspaces.data.path)/$(params.internalRequestResultsFile)")"
+        echo "$indexImage" | tr -d "\n" | tee "$(results.indexImage.path)"
```

Comment:
I think `jq -r` above will get rid of the new line for you, no?

Reply:
That is correct, Martin. I will get rid of these lines when  e2e is fixed. There is a ticket for it. 
Yes, Johnny. I added the "`-r `"  and forgot to remove the `tr`.

Reply:
fixed.

Reply:
@johnbieren , I had to add the `tr` back. There is a extra "\n" in the result itself, probably coming from the internal request.

Reply:
Last comment. Went  for the simpler version, as the one you approved. But please take the last look.

Reply:
Small detail: You don't need `-e` with `echo`. AFAIK, `-e` enables special characters and `-n` skips the newline at the end (that `echo` normally adds). Also, why `tee -a`? `-a` is for append, but we don't write to the result file anywhere above, so it won't make a difference.